content/en/04-tutorials/03-integrations/02-grafana.md

```diff
@@ -45,186 +45,215 @@ Before you can create a dashboard, you will need to add Pixie as a datasource:
+::: div image-m
+<svg title='' src='grafana/import-dashboard.png'/>
+:::
 
-5. Click the **Save** icon in the top right corner of your screen to save the dashboard.
+2. Upload a dashboard JSON file or paste the dashboard JSON text directly into the text area. The JSON files for Pixie's dashboards are located [here](https://github.com/pixie-io/grafana-plugin/tree/main/dashboards).
 
-::: div image-xl
-<svg title='' src='datasource-selector.png'/>
+::: div image-l
+<svg title='' src='grafana/import-dashboard-page.png'/>
 :::
 
-### Time series graph of HTTP throughput
+### Discover Pixie dashboards on Grafana.com
+
+Find Pixie dashboards at [Grafana.com/dashboards](https://grafana.com/dashboards).
+
+(coming soon)
+
+## Create a dashboard panel of Pixie data
 
-1. Copy & paste the following query into the Query Editor.
+First, we'll need to create a new dashboard:
 
-```python
-# Import Pixie's module for querying data.
-import px
+1. Click the **New dashboard** item under the **Dashboards** icon in the side menu.
 
-# Load data from Pixie's `http_events` table into a Dataframe.
-df = px.DataFrame(table='http_events', start_time=__time_from)
+2. In the new dashboard view, click **Add an empty panel**.
 
-# Bin the 'time_' column using the interval provided by Grafana.
-df.timestamp = px.bin(df.time_, __interval)
+### Select the Pixie datasource for your query
 
-# Group data by unique 'timestamp' and count the total number of
-# requests per unique timestamp.
-per_ns_df = df.groupby(['timestamp']).agg(
-        throughput_total=('latency', px.count)
-    )
+1. In the Edit Panel view, go to the **Query** tab.
 
-# Calculate throughput by dividing # of requests by the time interval.
-per_ns_df.request_throughput = per_ns_df.throughput_total / __interval
+2. Configure the query by selecting the **Pixie Grafana Datasource Plugin** from the data source selector.
 
-# Rename 'timestamp' column to 'time_'. The Grafana plugin expects a 'time_'
-# column to display data in a Graph or Time series.
-per_ns_df.time_ = per_ns_df.timestamp
-per_ns_df.request_throughput = per_ns_df.request_throughput * 1e9
+3. Click the **Save** icon in the top right corner of your screen to save the dashboard.
 
-# Output select columns of the DataFrame.
-px.display(per_ns_df['time_', 'request_throughput'])
-```
+::: div image-xl
+<svg title='' src='grafana/datasource-selector.png'/>
+:::
 
-This plugin uses the Pixie Language ([PxL](/reference/pxl)) to query telemetry data collected by the Pixie platform.
+### Add the `clusterName` variable
 
-The above PxL query outputs a table of timeseries data showing overall HTTP request throughput. Request throughput is calculated by counting the number HTTP requests that Pixie automatically traces in your cluster.
+The Pixie Grafana datasource plugin requires a `clusterName` dashboard variable. To create a dashboard new variable:
```

Comment:
It doesn't actually require the dashboard variable, it'll use the default cluster ID that you specified at plugin configuration.